doc/source/changes/version_0_22.rst.inc

```diff
@@ -208,3 +218,16 @@ Fixes
 
 * fixed array creation functions not raising an exception when called with wrong syntax func(axis1, axis2, ...)
   instead of func([axis1, axis2, ...]) (closes :issue:`203`).
+
+* fixed adding new sheets in a workbook: a new sheet is appended instead of prepended (closes :issue:`229`).
+
+* fixed __setitem__ of Workbook when value is an instance of `Sheet` and key is new.
+
+* fixed Workbook behavior in case of new workbook: the first added sheet replaces the default sheet `Sheet1`
+  (closes :issue:`230`).
+
+* fixed load session: read arrays in the same order as they are stored.
```

Comment:
something like: "fixed order of arrays in sessions loaded from files. The order is now the same as in the file"?